experimental/packages/exporter-logs-otlp-proto/README.md

```diff
@@ -0,0 +1,114 @@
+# OpenTelemetry Collector Exporter for node with protobuf
```

Comment:
I suggest using 'An OTLP exporter for logs using protobuf over HTTP'

Reply:
Changing